### PR TITLE
Added Oxygen orbs sound

### PR DESCRIPTION
**Pull request by xxfast**
You’ll hear me slurping :P

### CODE REVIEW

**Comment by xxfast** [comment]
[Environment Mechanic; Oxygen collection sound](https://trello.com/c/UL6zqLgg/54-environment-mechanic-oxygen-collection-sound)

File: Assets/Scripts/Audio/PlayerSFXController.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(AudioSource))]
+public class PlayerSFXController : SoundEffectsController {
+
+	public List<AudioClip> oxygenSounds;
```
Comment (6819710) [inline Assets/Scripts/Audio/PlayerSFXController.cs]: Audio Clips are fundementally differnt than audio source, I will be altering audio in the next sprint and cannot use scrips that directly play an aduio clip

Comment (xxfast) [reply Assets/Scripts/Audio/PlayerSFXController.cs]: read the base class implementation james, its playing the audio source not the audio clip

Comment (6819710) [reply Assets/Scripts/Audio/PlayerSFXController.cs]: Main issue here is that different audio clips may need different alterations made, by switching out the clip on a base source means that I cannot apply alterations from sound to sound, each sound should have its own audio channel (mixing), you should never batch apply effect and alterations.

File: Assets/Scripts/Audio/SoundEffectsController.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			source = value;
+		}
+	}
+
+	void Start () {
+		source = GetComponent<AudioSource> ();
+	}
+
+	public void Play(AudioClip toPlay){
+		Source.PlayOneShot (toPlay);
```
Comment (6819710) [inline Assets/Scripts/Audio/SoundEffectsController.cs]: This is playing a given audio clip on a given audio source, its the audio sources that are altered not the audio clip.

Comment (xxfast) [reply Assets/Scripts/Audio/SoundEffectsController.cs]: audio clips are indeed altered, try it out

File: Assets/Scripts/Audio/PlayerSFXController.cs (hunk 1 above)
Comment (xxfast) [reply Assets/Scripts/Audio/PlayerSFXController.cs]: why not to batch apply effects?

File: Assets/Scripts/Audio/SoundEffectsController.cs (hunk 2 above)
Comment (6819710) [reply Assets/Scripts/Audio/SoundEffectsController.cs]: the point I was making is that this is what is able to be altered. not the clips themselves.

File: Assets/Scripts/Audio/PlayerSFXController.cs (hunk 1 above)
Comment (6819710) [reply Assets/Scripts/Audio/PlayerSFXController.cs]: Differnt clips will require differnt alterations, you do apply batch effects in a parent channel, but you need to apply differnt alterations to make those batch effect to work correctly, you are talking to an audiophile. I have bothe the experiance and equipment to do this properly.

Comment (xxfast) [reply Assets/Scripts/Audio/PlayerSFXController.cs]: Will those alterations be on runtime? I don't think so. It already works correctly, this is not an issue that needs any alteration.

**Pull request merged**